src/components/pages/landing/header/index.tsx

```diff
@@ -29,22 +40,20 @@ const Header: React.FC<{topic?: string; customer?: any}> = ({
     setIsMounted(true)
   }, [])
 
-  let Offer = () => (
-    <>
-      <CreateAccount location="homepage header" />
-      <p className="max-w-sm pt-10 text-xs text-center sm:text-sm opacity-60">
-        Enter your email to create an account and start learning from more than
-        3,000 free lessons on egghead.
-      </p>
-    </>
-  )
+  const userPresent = Boolean(viewer || customer)
```

Comment:
clarified the logic a bit with these named variables nad made sure it was a boolean